terraform/tutorial.md

```diff
@@ -0,0 +1,351 @@
+# Cloud Operations Sandbox Tutorial
```

Comment:
Should this really be in the terraform directory? Seems like `/docs` could be a better place. Or `/`, or something else

Reply:
It would make more sense for it to be in `docs/`, but it has to be in the "current" directory when the user opens Cloud Shell, which is the `terraform/` directory for install script.